src/app/components/UpcomingConcerts/UpcomingConcerts.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import moment from 'moment'
+import styled from 'styled-components'
+
+type Concert = {
+  id: string
+  mainAct: string
+  support?: string
+  concertDate: string
+  location: string
+  numberOfTickets: number
+}
+
+type UpcomingConcertsProps = {
+  concerts: Concert[]
+}
+export default function UpcomingConcerts({
+  concerts,
+}: UpcomingConcertsProps): JSX.Element {
+  const numberOfUpcomingConcerts = concerts.filter((concert) =>
+    moment(concert.concertDate).isSameOrAfter(moment())
```

Comment:
`moment` is a very strong and big library, which heavily increases your bundle size.
If you plan to use more moment functions, it's a great lib. But if you have only some use-case, I would recommend implementing a solution based on https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date

It could look like this in vanillajs.
```suggestion
    new Date(concert.concertDate) > new Date()
```